src/units/unit.hpp

```diff
@@ -1786,6 +1786,12 @@ class unit : public std::enable_shared_from_this<unit>
 	 */
 	bool ability_matches_filter(const config & cfg, const std::string& tag_name, const config & filter) const;
 
+	// Return a string used for create a second halo for unit under "halos" ability if active
+	const std::string halo_abilities() const;
+
+	// Return a vector string used for create a second set of overlays for unit under "overlays" ability if active
```

Comment:
I'd prefer to have this near the `overlays()` function, instead of being nested within the abilities. As for documentation, maybe

List of the active abilities' overlay= attributes.